watson-developer-cloud/discovery-components
# chore: add unit test for documentPreview utils

#### What do these changes do/fix?
Adds unit tests to the documentPreview utils file `documentData.ts` for higher test coverage of our code.

https://zenhub.ibm.com/app/workspaces/discovery-unified-5728bf39d2c20f776e05d0ef/issues/watson-discovery/docviz-squad-issue-tracker/234

#### How do you test/verify these changes?
`yarn test watch:documentData` and see that tests pass.

Run `yarn test:coverage` and view the coverage of the `DocumentPreview/utils/documentData.ts` file to see the results are statements: 100%, branches: 100%, and functions: 100%.
